templates/field/field--node--oe-consultation-response-button--oe-consultation--full.html.twig

```diff
@@ -0,0 +1,21 @@
+{#
+/**
+ * @file
+ * Respond button link field in the Consultation content full view mode.
+ *
+ * @see ./core/themes/stable/templates/field/field.html.twig
+ */
+#}
+{% set label = 'Respond to the questionnaire'|t %}
+{% if items.0.content['#url'].uri != items.0.content['#title'] %}
+  {% set label = items.0.content['#title'] %}
+{% endif %}
+<div class="ecl-u-mt-m">
+  {% include '@ecl-twig/link' with {
+    link: {
+      type: 'cta',
+      label: label,
+      path: items.0.content['#url'],
+    }
+  } %}
+</div>
```

Comment:
I could save `<front>` and `<front>` in the url field but it produces an error when it's being rendered.
![image](https://user-images.githubusercontent.com/22004498/105712572-44068b00-5f1a-11eb-8374-e72d75544123.png)
![image](https://user-images.githubusercontent.com/22004498/105712598-4f59b680-5f1a-11eb-86f2-0c711264d138.png)
![image](https://user-images.githubusercontent.com/22004498/105712761-8f209e00-5f1a-11eb-8cee-cff94e7be95b.png)
`path: items.0.content['#url'],` is not okay to use directly here.